packages/stories/package.json

```diff
@@ -17,6 +17,7 @@
     "build": "build-storybook"
   },
   "dependencies": {
+    "@gitgraph/core": "1.4.5",
```

Comment:
Oh you're right, we depend directly on the Core package. See https://github.com/nicoespeon/gitgraph.js/blob/884f8496fcf754217ca7e067820ed71b4b88693d/packages/stories/src/gitgraph-react/1-basic-usage.stories.tsx#L4

Good catch 👍